docs/dev/android/setup-android-build-environment.md

```diff
@@ -4,12 +4,12 @@
 
 This document describes how to make local builds of the Android bindings in
 this repository. Most consumers of these bindings *do not* need to follow
-this process, but will instead use pre-built bindings [todo: link to this]
+this process, but will instead use pre-built bindings [TODO: link to this]
```

Comment:
huh? we should ... add the link (or file a bug)

Reply:
I guess that should go here? https://github.com/mozilla/glean/releases  Or link to the wrapper in a-c?

Reply:
Or maybe just link to [our other docs](https://mozilla.github.io/glean/book/user/adding-glean-to-your-project.html) ?